common-libs/did-auth-lib/README.md

```diff
@@ -1,36 +1,193 @@
 # Affinidi Did Auth Helpers (PUBLIC)
```

Comment:
@oleksiipiliugin why we mention in a headers that helpers is public ?

Reply:
I think it comes here from GitLab where we had such flags to mark if the library is public or private.